HomeWork2/DmytroFenevHW2.cpp

```diff
@@ -0,0 +1,159 @@
+#include <cmath>
+#include <iostream>
+#include <string>
+
+int main() {
+  int z;
+  std::cout << "Zadacha №1, N2, N3, N4, N5\n";
+  std::cin >> z;
+  int x, xp, xp1;
+  int y, y1, yp;
+  int n, np, nc;
+  int sum, sum1, sum2;
+  int l2;
+  int k2;
+  double k3;
+  int sample[10] = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
```

Comment:
явная инициализация 0 не есть объязательная
```suggestion
  int sample[10] {};
```

Reply:
c изменениями согласен